specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2022-02-01-preview/common.json

```diff
@@ -0,0 +1,94 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-02-01-preview",
+    "title": "KeyVaultManagementClient",
+    "description": "The Azure management API provides a RESTful set of web services that interact with Azure Key Vault."
+  },
+  "paths": {},
+  "definitions": {
+    "CloudError": {
+      "description": "An error response from Key Vault resource provider",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/CloudErrorBody"
+        }
+      },
+      "x-ms-external": true,
+      "type": "object"
+    },
+    "CloudErrorBody": {
+      "description": "An error response from Key Vault resource provider",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "Error code. This is a mnemonic that can be consumed programmatically."
+        },
+        "message": {
+          "type": "string",
+          "description": "User friendly error message. The message is typically localized and may vary with service version."
+        }
+      },
+      "x-ms-external": true,
+      "type": "object"
+    },
+    "SystemData": {
```

Comment:
>SystemData [](http://example.com/codeflow?start=5&length=10)

If you change to referencing the systemData definition in common-types/v4 would it be non-breaking? Can you do that instead?

Reply:
@TimLovellSmith Quick background regarding these changes:

- The changes to the KeyVault files were made to fix the S360 issues related to swagger completeness.
- I had created a dev branch out of the main branch, then created another branch from dev branch. Then created a PR from my branch to dev branch: https://github.com/Azure/azure-rest-api-specs/pull/22012.
- I had fixed all the errors reported in the CI pipeline stages including the breaking changes. There was one stage [SDK azure-powershell](https://github.com/Azure/azure-rest-api-specs/pull/22012/checks?check_run_id=10427914861) which was failing, but the Swagger team confirmed that the issue was not related to my changes.
- Could you please help me understand, why I am running into new undetected errors in this CI pipeline, which were undetected in the earlier PR to dev branch?